spec/peep_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'peep'
+
+describe Peep do
+  it 'list all the peeps' do
+    connection = PG.connect(dbname: 'chitter_test')
+
+    connection.exec("INSERT INTO peeps VALUES (1, 'First peep');")
+    connection.exec("INSERT INTO peeps VALUES (2, 'Second peep');")
+    connection.exec("INSERT INTO peeps VALUES (3, 'Third peep');")
+
+    peeps = Peep.all
+
+    expect(peeps).to include("First peep" && "Second peep" && "Third peep")
```

Comment:
You could make this test stronger. You know exactly what list you would expect to be returned because you know exactly what's in the database so using `include` instead of `eq` is too permissive.